src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -22,42 +22,42 @@ object BooleanOperatorsSpecification extends Properties("Boolean Operators"):
 
   property("and") = forAll { (pair: (Boolean, Boolean)) =>
     val (left, right) = pair
-    
+
     and(left, right) == left && right
```

Comment:
If you run the test you see that it fails. The same for `or`.

Moreover I asked to add an extra test for "eagerness". The same of `or`.